docs/releasing/publishing.md

```diff
@@ -52,30 +52,18 @@ Developers should pair on releases. When remote working, it can be useful to be
 
 7. Save the changes. Do not commit.
 
-8. Run `npm run build-release`. You will be now be prompted to continue or cancel.
+8. Run `npm run build-release` to:
 
-9. (Optional) Test in [GOV.UK Design System](git@github.com:alphagov/govuk-design-system.git)
+- build GOV.UK Frontend into the `/package` and `/dist` directories
+- commit the changes
+- push a branch to GitHub
 
-  If you want to test that your changes work in the GOV.UK Design System, you can use [npm link](https://docs.npmjs.com/cli/link) to test before publishing.
+  You will be now be prompted to continue or cancel.
 
-  ```bash
```

Comment:
@vanitabarrett Do we need to remove similar content from [steps 9 and 10 of the 'Build a new release' section](https://github.com/alphagov/govuk-frontend/blob/main/docs/releasing/publishing-from-a-support-branch.md#build-a-new-release) in the 'Publishing from a support branch' doc?

Reply:
Similar to lines 61-72 here, I mean, not just line 61! :)

Reply:
@EoinShaughnessy Yes, I think we should make the same changes that we've made in this PR where we can in the 'Publishing from a support branch' doc.

In the call yesterday, we discussed that the documentation we're removing here is still useful for developers, just not during the release process. We're more likely to need this documentation when we want to test a local change in GOV.UK Frontend against the Design System. 

I wonder if it would be useful to include in some other existing documentation file, e.g: "testing" or "running locally".

Reply:
That sounds like a good idea! I'll try including it in 'Testing'.